exp-player/addon/styles/components/isp-radio-group.scss

```diff
@@ -24,6 +24,11 @@
   margin-top: -15px;
 }
 
-.label-spacing {
-  max-width: 6%;
+.measure-four {
+  width: 6%;
```

Comment:
Can you please clarify how you arrived at this number?  These styles seem to be very question-specific. If this is to label something reused repeatedly (eg 7 point scale), you may not want the style to be tied to just one question on a form.